libcontainer/specconv/spec_linux.go

```diff
@@ -497,6 +497,18 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 	return config, nil
 }
 
+func toConfigIDMap(specMaps []specs.LinuxIDMapping) []configs.IDMap {
```

Comment:
I think it makes sense to also switch `setupUserNamespace` to using this function. This may be done by a separate commit just before this one.

Reply:
Good call, switched that. Thanks